package.json

```diff
@@ -1,25 +1,35 @@
 {
-	"name": "vue2-vite",
-	"version": "0.0.0",
-	"scripts": {
-		"json-server": "json-server src/api/db.json --port 3004",
-		"dev": "vite",
-		"build": "vite build",
-		"serve": "vite preview",
-		"start": "vue-cli-service serve"
-	},
-	"dependencies": {
-		"json-server": "^0.16.3",
-		"vee-validate": "^2.2.15",
-		"vite-plugin-vue2": "^1.4.0",
-		"vue": "^2.6.12",
-		"vue-types": "^2.0.3"
-	},
-	"devDependencies": {
-		"@rollup/plugin-eslint": "^8.0.1",
-		"@vue/cli-service": "^4.5.12",
-		"eslint": "^7.27.0",
-		"eslint-plugin-vue": "^7.9.0",
-		"vite": "^2.0.5"
-	}
+  "name": "vue2-vite",
+  "version": "0.0.0",
+  "scripts": {
+    "json-server": "json-server src/api/db.json --port 3004",
+    "dev": "vite",
+    "build": "vite build",
+    "test": "vue-cli-service test:unit",
+    "serve": "vite preview",
+    "start": "vue-cli-service serve"
+  },
+  "dependencies": {
+    "jest": "^27.0.3",
+    "json-server": "^0.16.3",
+    "vee-validate": "^2.2.15",
+    "vite-plugin-vue2": "^1.4.0",
+    "vue": "^2.6.12",
+    "vue-types": "^2.0.3",
+    "vuex": "^3.6.2"
+  },
+  "devDependencies": {
+    "@rollup/plugin-eslint": "^8.0.1",
+    "@testing-library/jest-dom": "^5.12.0",
+    "@testing-library/vue": "^5.6.2",
+    "@types/jest": "^26.0.23",
+    "@vue/babel-preset-app": "^4.5.13",
+    "@vue/cli-plugin-unit-jest": "^4.5.13",
+    "@vue/cli-service": "^4.5.12",
+    "@vue/test-utils": "^1.2.0",
```

Comment:
You shouldn't need this dependency explicitly if you are using only `testing-library`. 🤔